dwave/system/testing.py

```diff
@@ -114,19 +114,42 @@ class MockDWaveSampler(dimod.Sampler, dimod.Structured):
     edgelist = None
     properties = None
     parameters = None
-
-    # by default, use ExactSolver for problems up to size (inclusive):
+    # In principle this class can be wrapped and additional parameters emulated
+    # (see shimming-tutorial)
+    
+    # by default, use ExactSolver to determine the first sample up to size (inclusive):
     EXACT_SOLVER_CUTOFF_DEFAULT = 16
 
     def __init__(self,
                  nodelist=None, edgelist=None, properties=None,
                  broken_nodes=None, broken_edges=None,
                  topology_type=None, topology_shape=None,
                  parameter_warnings=True,
+                 mocking_sampler = None, # Add default sampler
```

Comment:
[double space for inline comments](https://peps.python.org/pep-0008/#:~:text=Inline%20comments%20should%20be%20separated,a%20%23%20and%20a%20single%20space.)

and no spaces surrounding `=` sign

Reply:
TIL [scroll-to-text](https://wicg.github.io/scroll-to-text-fragment/) draft standard. Neat!